.gitignore

```diff
@@ -25,13 +25,7 @@ public/client
 public/system
 public/uploads
 
-plugins/*/
-plugins/Gemfile
-!plugins/*.rb
-!plugins/*.yml
-!plugins/*.md
-# NB: To reveal a plugin to hack on it locally, add a line here telling .gitignore to reveal it:
-# !plugins/my_awesome_plugin
+plugins/
```

Comment:
At least for me I need to gitignore because otherwise it doesn't show up when I use Atom to navigate to files or search for them using cmd+P